BenGWeeks/Zapp.ie
# Bug 95671: Send Zap Adaptive Card remains Open

### Description

After sending the Sats, the adaptive card gets blocked, and only information about the sent Sats remains.

Two bugs in one PR, because logic is connected to both of them:

1. To send the next Sats to another person, we need to create a new request, as the previous adaptive card is blocked (Bug 95671)
2. The adaptive card remains blocked, and information about the sent Sats (Receiver, Message, and Amount) stays visible (Bug 95994)

### Screenshot/video

![image](https://github.com/user-attachments/assets/f7deda77-f4b7-4cf3-a843-db761ca76ebc)

### Related workitems

- [Bug 95671](https://dev.azure.com/EvrosPowerPlatformTeam/Default/_workitems/edit/95671): Send Zap Adaptive Card rains Open
- [Bug 95994](https://dev.azure.com/EvrosPowerPlatformTeam/Default/_workitems/edit/95994): Previous zap info is lost

Fixes # (issue)

### Type of Change


### Test Instructions

To test this PR:

1. Open Zapp.ie (local) bot
2. Select prompt "Send Zap"
3. Fill out the form and submit the request.
4. Check if the adaptive card is blocked and ensure that the correct information remains visible.

### Checklist:
